core/domain/story_services.py

```diff
@@ -174,6 +175,14 @@ def apply_change_list(story_id, change_list):
                 elif (change.property_name ==
                       story_domain.STORY_PROPERTY_THUMBNAIL_FILENAME):
                     story.update_thumbnail_filename(change.new_value)
+                    thumbnail_url = (
+                        'http://localhost:8181/assetsdevhandler/story/%s'
+                        '/assets/thumbnail/%s'
+                        % (story_id, change.new_value))
+                    thumbnail_size_in_bytes = python_utils.url_open(
+                        thumbnail_url).headers['Content-Length']
```

Comment:
need suggestion on this to do in a better way and for production.

Reply:
Check comment above. Probably getting the size in bytes should be added as a function to the classes in fs_domain.

Reply:
done